vsintegration/src/FSharp.ProjectSystem.FSharp/xlf/VSPackage.es.xlf

```diff
@@ -433,12 +433,12 @@
         <note />
       </trans-unit>
       <trans-unit id="ProductDetails">
-        <source>Microsoft Visual F# 4.3</source>
+        <source>Microsoft Visual F# 10.1 for F# 4.1</source>
         <target state="needs-review-translation">Microsoft Visual F# 4.1</target>
         <note />
       </trans-unit>
       <trans-unit id="9002">
-        <source>Microsoft Visual F# 4.3</source>
+        <source>Microsoft Visual F# 10.1 for F# 4.1</source>
```

Comment:
Perhaps the text ``Visual F# 10.1`` should be universally replaced by ``Visual F# Tools 10.1``  just to clarify?

Reply:
@dsyme 

That table doesn't work quite the way you think it does:
this is the list of redirects it will produce:

````
      <dependentAssembly>
        <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
        <bindingRedirect oldVersion="2.0.0.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="2.3.0.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="2.3.5.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="4.0.0.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="4.3.0.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.3.1.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="2.3.5.1" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.78.3.1" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.259.3.1" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="4.3.1.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.47.4.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.78.4.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.259.4.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="4.4.0.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.47.41.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.78.41.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="3.259.41.0" newVersion="4.4.3.0" />
        <bindingRedirect oldVersion="4.4.1.0" newVersion="4.4.3.0" />
      </dependentAssembly>
````

Note that 4.0 will correctly be redirected:

<bindingRedirect oldVersion="4.4.0.0" newVersion="4.4.3.0" />
There is no reason for a 4.3 line, because there are no portable profiles.

Sure I can update the text to Visual F# Tools 10.1

Reply:
ok, thanks!